README.md

```diff
@@ -1 +1,52 @@
-# configure
\ No newline at end of file
+# configure
+
+Easily fill in a configuration struct with either flag arguments or a JSON blob argument.
+
+
+## usage
+
+Given a simple program that uses two arguments, `district_id` and `collection`:
+
+```go
+var config struct {
+	DistrictID string `config:"district_id,required"`
+	Collection string `config:"collection"`
+}
+
+func main() {
+	if err := configure.Configure(&config); err != nil {
```

Comment:
What are your thoughts on calling this method `Parse` instead of `Configure`? Seems truer to what it does and would read nicely